src/DynamoCoreWpf/ViewModels/Search/BrowserItemViewModel.cs

```diff
@@ -152,6 +153,30 @@ public string Assembly
             }
         }
 
+        public ElementTypes ElementType
+        {
+            get
+            {
+                if (entries.Any(entry => entry.ElementType.HasFlag(ElementTypes.BuiltIn)) ||
+                    subCategories.Any(subCat => subCat.ElementType.HasFlag(ElementTypes.BuiltIn)))
+                    return ElementTypes.BuiltIn;
+
+                if (entries.All(entry => entry.ElementType.HasFlag(ElementTypes.Packaged)) &&
+                    subCategories.All(subCat => subCat.ElementType.HasFlag(ElementTypes.Packaged)))
+                    return ElementTypes.Packaged;
+
+                if (entries.All(entry => entry.ElementType.HasFlag(ElementTypes.ZeroTouch)) &&
+                    subCategories.All(subCat => subCat.ElementType.HasFlag(ElementTypes.ZeroTouch)))
+                    return ElementTypes.ZeroTouch;
+
+                if (entries.All(entry => entry.ElementType.HasFlag(ElementTypes.CustomNode)) &&
+                    subCategories.All(subCat => subCat.ElementType.HasFlag(ElementTypes.CustomNode)))
+                    return ElementTypes.CustomNode;
+
+                return ElementTypes.None;
+            }
+        }
+
```

Comment:
You are right... I tried. Element type called MUCH frequently then Collection Changed.
![image](https://cloud.githubusercontent.com/assets/8158404/7366547/d68ea8f4-eda1-11e4-9dec-5e0cf8529f44.png)
First number is  ElementTypeCalled, second number is CollectionChanged.